dynflow.gemspec

```
@@ -24,18 +24,23 @@ Gem::Specification.new do |s|
   s.add_dependency "apipie-params"
   s.add_dependency "concurrent-ruby", '~> 1.1.3'
   s.add_dependency "concurrent-ruby-edge", '~> 0.6.0'
+  s.add_dependency "csv", "~> 3.3"
```

https://stdgems.org/csv/ says Ruby 2.7 used csv 3.1.2 so perhaps widen it a bit?
```suggestion
  s.add_dependency "csv", "~> 3.1"
```